docker-compose.yml

```diff
@@ -10,10 +10,12 @@ services:
     image: docker.artifactory.reform.hmcts.net/fpl/fpl-service
     container_name: fpl-service
     environment:
+      - IDAM_API_URL=http://idam-api:8080
       - DOCUMENT_MANAGEMENT_URL=http://dm-store:8080
       - CORE_CASE_DATA_API_URL=http://ccd-data-store-api:4452
       - IDAM_S2S_AUTH_URL=http://service-auth-provider-api:8080
       - IDAM_S2S_AUTH_TOTP_SECRET=AABBCCDDEEFFGGHH
+      - FPL_LOCAL_AUTHORITY_MAPPING=example.com=>example
```

Comment:
Let's rename this to be `FPL_LOCAL_AUTHORITY_NAME_MAPPING` to make it less generic.